src/harness/reference_models/iap/test_data/gwpz_1.json

```diff
@@ -0,0 +1,54 @@
+{  
+   "id":"incumbent/uls/1",
+   "type":"PART_90",
+   "deploymentParam":{  
+      "operationParam":{  
+         "operationFrequencyRange":{  
+            "lowFrequency":3680000000.0,
+            "highFrequency":3690000000.0
```

Comment:
Shouldn't the GWPZ bandwidth be either 25MHz or 50 MHz?

Reply:
Updated   to 50MHz